apps/frontpage/components/home/hero/index.tsx

```diff
@@ -55,15 +55,32 @@ const Star = ({ x = 0, y = 0, w = 14, delay = 0 }) => {
 
 export const Hero = () => {
   const [slide, setSlide] = useState(1);
+  const intervalId = useRef<NodeJS.Timeout | null>(null);
 
-  useEffect(() => {
-    const interval = setInterval(() => {
+  const setSlideInterval = () => {
+    if (intervalId.current) {
+      clearInterval(intervalId.current);
+    }
+
+    intervalId.current = setInterval(() => {
       setSlide((slide) => (slide === 4 ? 1 : slide + 1));
     }, 3000);
+  };
 
-    return () => clearInterval(interval);
+  useEffect(() => {
+    setSlideInterval();
+    return () => {
+      if (intervalId.current) {
+        clearInterval(intervalId.current);
+      }
+    };
   }, []);
 
+  const handleSlideChange = (newSlide: number) => {
```

Comment:
I expected it to stop automatically switching between the four tabs once I clicked on one. Wdyt?

Reply:
To stop automatically? If you click it reset the timer and will start again after 3 seconds. I don't think we should make it stop forever.

Reply:
To stop automatically? If you click it reset the timer and will start again after 3 seconds. I don't think we should make it stop forever.

Reply:
After some thoughts and discussion you were right. It will look better if we stopped the animation if the user clicks on it. I'll change that.